### PR TITLE
Multiple issues with read-only mode (MBS-8943)

### PR DESCRIPTION
**Pull request by uklauer**
The read-only phase revealed some issues with the read-only mode. See the individual commits for details.

### CODE REVIEW

**Comment by uklauer** [comment]
I branched off from `production`, so it should be possible to deploy it even before the schema change if desired.

File: lib/MusicBrainz/Server/Controller/CDTOC.pm [hunk 1]
```diff
@@ -139,7 +139,7 @@ sub set_durations : Chained('load') PathPart('set-durations') Edit
     );
 }
 
-sub attach : Local DenyWhenReadOnly
+sub attach : Local DenyWhenReadonly
```
Comment (uklauer) [inline lib/MusicBrainz/Server/Controller/CDTOC.pm]: Yes, it’s case-sensitive.

File: lib/MusicBrainz/Server/Controller/WS/js/Edit.pm [hunk 2]
```diff
@@ -578,6 +582,9 @@ sub edit : Chained('/') PathPart('ws/js/edit') CaptureArgs(0) Edit {
     unless ($c->user->has_confirmed_email_address) {
         $c->forward('/ws/js/detach_with_error', ['a confirmed email address is required']);
     }
+    if ($c->user->is_editing_disabled) {
+        $c->forward('/ws/js/detach_with_error', ['you are not allowed to enter edits']);
```
Comment (uklauer) [inline lib/MusicBrainz/Server/Controller/WS/js/Edit.pm]: This is not related to read-only mode, but was another thing not in sync with the regular edit pages.

**Comment by mwiencek** [comment]
Thanks! :)

**Pull request merged**